### ISSUE TITLE
Initial docker-compose build issue

### ISSUE THREAD

**Issue by Elevennails** [pwndoc/pwndoc#99]
I'm a noob with docker so sorry if its me! 

When i run docker compose i get to Step 6/11 and the process seems to stop. I've checked the repo for "https://github.com/kelektiv/node.bcrypt.js/releases/download/v3.0.8/bcrypt_lib-v3.0.8-node-v83-linux-x64-musl.tar.gz " which is 404ing and it appear to have changed structure could this be the problem. I know the response is only a warning but the install does not progress from this point for me.

Step 6/11 : RUN npm install
 ---> Running in 7304588a88b1

> bcrypt@3.0.8 install /app/node_modules/bcrypt
> node-pre-gyp install --fallback-to-build

node-pre-gyp WARN Using request for node-pre-gyp https download 
node-pre-gyp WARN Tried to download(404): https://github.com/kelektiv/node.bcrypt.js/releases/download/v3.0.8/bcrypt_lib-v3.0.8-node-v83-linux-x64-musl.tar.gz 
node-pre-gyp WARN Pre-built binaries not found for bcrypt@3.0.8 and node@14.16.0 (node-v83 ABI, musl) (falling back to source compile with node-gyp)

**Comment by yeln4ts** [comment]
can you paste the full log ?

**Comment by Elevennails** [comment]
Looks like your last commit has resolved the issue. I've done another full refresh a git pull and rebuild and it's continued past Step 6/11,  where is was stopping. see below. Appears resolved will eval monday.

Thanks


```
user@user-Virtual-Machine:/opt/pwndoc$ docker-compose down
Removing network pwndoc_backend
user@user-Virtual-Machine:/opt/pwndoc$ docker images
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
<none>              <none>              7aa61b4754d4        10 hours ago        117MB
mongo               latest              bbfd3e575f12        41 hours ago        449MB
node                lts-alpine          e2cefec76a61        9 days ago          116MB
user@user-Virtual-Machine:/opt/pwndoc$ docker image rm bbfd3e575f12
Untagged: mongo:latest
Untagged: mongo@sha256:51f6fdbfc622d91e276ade7e6cf6491aa36ff2bd9b158dadb732f9e4a05f33ad
Deleted: sha256:bbfd3e575f124286c63a43df117c98f76d8c77e7aa8b0c85ab366d50964bf9f8
Deleted: sha256:f8128e7f9277d1b6ec68f82f5eebec6c821b688aa6aa96a9bab7351a1d8a3b32
Deleted: sha256:45120c097ca2a5f388e6d7a8c2cd47c5fb0c3cf46b87778ef9f03f3e536eac1a
Deleted: sha256:a8b8534508981c89e69ac2768fd8d636f4167a64dbeb490e65142035b1438518
Deleted: sha256:dde04d378e126dd69060c595f978836eada68a21fe45698815e83aa349ec4a0a
Deleted: sha256:fbc935557976404c729a6f896dbfbc4857e843d110778e1c8bb6ec2d5f6c1f28
Deleted: sha256:443f368883120b5aae75ec9669333f8a77618c17ed462f0ffad310fa5aded966
Deleted: sha256:8ef442d47f406a123a2f8e7559ae6d6149b2f53dd26cbfef4514311d5f4820d1
Deleted: sha256:027f4973d3afbe66b83d4e6ab3c1eccc5fc9b64a7b9e6fc58097492a912fbaf1
Deleted: sha256:d9dddb50d43ea5f15b47b9aefd94bdfb29c085db5c6b568e3892fe591dfb8add
Deleted: sha256:cbcdeed56d9b86e4d36dd5e4e68aae7a8544b9ce13302ba9f29981ea961c5e4a
Deleted: sha256:cff1beb2b6818fbc5e4800231f3558dd7da191e7124bc85d8c45560ed47a31da
Deleted: sha256:837d6facb613e572926fbfe8cd7171ddf5919c1454cf4d5b4e78f3d2a7729000
user@user-Virtual-Machine:/opt/pwndoc$ docker container ls -al
CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS                       PORTS               NAMES
ef63357b73a8        7aa61b4754d4        "/bin/sh -c 'apk --n…"   10 hours ago        Exited (255) 3 minutes ago                       quirky_montalcini
user@user-Virtual-Machine:/opt/pwndoc$ docker container rm ef63357b73a8
ef63357b73a8
user@user-Virtual-Machine:/opt/pwndoc$ docker image rm 7aa61b4754d4
Deleted: sha256:7aa61b4754d47b2706d5b9ccbbcc0fc457ed39fe74bc35112cbaea7e7270d539
Deleted: sha256:c166ef50e9e9ec4c5cb9c721928a3967a758c40fe3bfe85125f58b2113c14bad
Deleted: sha256:52fc129798b60bc05eaba68d5528ceb214cd5e5721b572080b6683b2a6c26394
Deleted: sha256:a6d6ee61321957f2af07bf2b993171799b1b863af558f2dc7ad1e155c2af6ebf
Deleted: sha256:8709d7beb0cafffce0f76128c632c1c84d76839cb74f909a5cdd9f0a30bc5da7
user@user-Virtual-Machine:/opt/pwndoc$ docker image rm e2cefec76a61
Untagged: node:lts-alpine
Untagged: node@sha256:76badf0d0284ad838536d49d8a804988b4985fc6bc7242dfff4f8216c851438b
Deleted: sha256:e2cefec76a616d84101ea518668da92b651e046bd9ff1b07c7607df5c4024558
Deleted: sha256:7fb0176f6a89010e7257b9eb1145dc56ad1d0e2ad8e0b72f0c6781af16cbe8ac
Deleted: sha256:537dd411e1faac5a25c4fbf781018e9dd35a98f77b528aa6223072b587333a49
Deleted: sha256:cc970d26f845506321e4c14b6c2cab984fb38afcc2c6089fb44bac61a1040b40
Deleted: sha256:6169bf830ae69f192382cfe9a6fe96e7f28a21c0c5e5045b85871b510de58435
user@user-Virtual-Machine:/opt/pwndoc$ docker images
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
user@user-Virtual-Machine:/opt/pwndoc$ git pull
remote: Enumerating objects: 41, done.
remote: Counting objects: 100% (41/41), done.
remote: Compressing objects: 100% (11/11), done.
remote: Total 41 (delta 30), reused 41 (delta 30), pack-reused 0
Unpacking objects: 100% (41/41), 8.96 KiB | 573.00 KiB/s, done.
From https://github.com/pwndoc/pwndoc
   9a2e402..3d1861b  master     -> origin/master
Updating 9a2e402..3d1861b
Fast-forward
 backend/src/lib/report-generator.js                |  15 +-
 backend/src/models/audit.js                        |   9 +-
 backend/src/models/custom-field.js                 |  23 +-
 backend/src/routes/audit.js                        |   3 +-
 backend/src/routes/data.js                         |  32 +-
 frontend/src/components/custom-fields.vue          |  89 +++--
 frontend/src/components/editor.vue                 |   1 -
 .../src/pages/audits/edit/findings/edit/edit.html  |   2 -
 .../src/pages/audits/edit/findings/edit/edit.js    |  64 ++--
 .../src/pages/audits/edit/general/general.html     |  17 +
 frontend/src/pages/audits/edit/general/general.js  |  59 +++-
 frontend/src/pages/datas/custom/custom.html        | 386 ++++++++++-----------
 frontend/src/pages/datas/custom/custom.js          |  67 +++-
 .../src/pages/vulnerabilities/vulnerabilities.html |  38 +-
 .../src/pages/vulnerabilities/vulnerabilities.js   |  42 +--
 frontend/src/services/utils.js                     |  41 +++
 16 files changed, 495 insertions(+), 393 deletions(-)
user@user-Virtual-Machine:/opt/pwndoc$ docker-compose up -d --build
Creating network "pwndoc_backend" with driver "bridge"
Pulling mongodb (mongo:)...
latest: Pulling from library/mongo
92dc2a97ff99: Pull complete
be13a9d27eb8: Pull complete
c8299583700a: Pull complete
f61ed17142e4: Pull complete
bed7676d225b: Pull complete
ba553bcfc69c: Pull complete
e5046b6c236f: Pull complete
80191acfded2: Pull complete
d41d63fc76cc: Pull complete
5605b8c2e9f7: Pull complete
e8b16825b485: Pull complete
3d40ccce1309: Pull complete
Digest: sha256:51f6fdbfc622d91e276ade7e6cf6491aa36ff2bd9b158dadb732f9e4a05f33ad
Status: Downloaded newer image for mongo:latest
Building pwndoc-backend
Step 1/11 : FROM node:lts-alpine
lts-alpine: Pulling from library/node
e95f33c60a64: Pull complete
26e60022b4e3: Pull complete
3735863698a8: Pull complete
d4cc1a999352: Pull complete
Digest: sha256:76badf0d0284ad838536d49d8a804988b4985fc6bc7242dfff4f8216c851438b
Status: Downloaded newer image for node:lts-alpine
 ---> e2cefec76a61
Step 2/11 : RUN mkdir -p /app
 ---> Running in 1effe9f3ec2c
Removing intermediate container 1effe9f3ec2c
 ---> 4a5ff9a78ebe
Step 3/11 : WORKDIR /app
 ---> Running in 0331c3721d28
Removing intermediate container 0331c3721d28
 ---> 52ca6c3c8ddc
Step 4/11 : COPY package*.json ./
 ---> 41a7efc64b33
Step 5/11 : RUN apk --no-cache add --virtual builds-deps build-base python
 ---> Running in fe165f67f7dd
fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/community/x86_64/APKINDEX.tar.gz
(1/26) Installing binutils (2.33.1-r0)
(2/26) Installing libmagic (5.37-r1)
(3/26) Installing file (5.37-r1)
(4/26) Installing gmp (6.1.2-r1)
(5/26) Installing isl (0.18-r0)
(6/26) Installing libgomp (9.3.0-r0)
(7/26) Installing libatomic (9.3.0-r0)
(8/26) Installing mpfr4 (4.0.2-r1)
(9/26) Installing mpc1 (1.1.0-r1)
(10/26) Installing gcc (9.3.0-r0)
(11/26) Installing musl-dev (1.1.24-r3)
(12/26) Installing libc-dev (0.7.2-r0)
(13/26) Installing g++ (9.3.0-r0)
(14/26) Installing make (4.2.1-r2)
(15/26) Installing fortify-headers (1.1-r0)
(16/26) Installing build-base (0.5-r1)
(17/26) Installing libbz2 (1.0.8-r1)
(18/26) Installing expat (2.2.9-r1)
(19/26) Installing libffi (3.2.1-r6)
(20/26) Installing gdbm (1.13-r1)
(21/26) Installing ncurses-terminfo-base (6.1_p20200118-r4)
(22/26) Installing ncurses-libs (6.1_p20200118-r4)
(23/26) Installing readline (8.0.1-r0)
(24/26) Installing sqlite-libs (3.30.1-r2)
(25/26) Installing python2 (2.7.18-r0)
(26/26) Installing builds-deps (20210305.215134)
Executing busybox-1.31.1-r9.trigger
OK: 218 MiB in 42 packages
Removing intermediate container fe165f67f7dd
 ---> 2b782b33923e
Step 6/11 : RUN npm install
 ---> Running in ae57aa846aa3

> bcrypt@3.0.8 install /app/node_modules/bcrypt
> node-pre-gyp install --fallback-to-build

node-pre-gyp WARN Using request for node-pre-gyp https download 
node-pre-gyp WARN Tried to download(404): https://github.com/kelektiv/node.bcrypt.js/releases/download/v3.0.8/bcrypt_lib-v3.0.8-node-v83-linux-x64-musl.tar.gz 
node-pre-gyp WARN Pre-built binaries not found for bcrypt@3.0.8 and node@14.16.0 (node-v83 ABI, musl) (falling back to source compile with node-gyp) 
make: Entering directory '/app/node_modules/bcrypt/build'
  CXX(target) Release/obj.target/bcrypt_lib/src/blowfish.o
  CXX(target) Release/obj.target/bcrypt_lib/src/bcrypt.o
  CXX(target) Release/obj.target/bcrypt_lib/src/bcrypt_node.o
In file included from ../src/bcrypt_node.cc:1:
../../nan/nan.h: In function 'void Nan::AsyncQueueWorker(Nan::AsyncWorker*)':
../../nan/nan.h:2298:62: warning: cast between incompatible function types from 'void (*)(uv_work_t*)' {aka 'void (*)(uv_work_s*)'} to 'uv_after_work_cb' {aka 'void (*)(uv_work_s*, int)'} [-Wcast-function-type]
 2298 |     , reinterpret_cast<uv_after_work_cb>(AsyncExecuteComplete)
      |                                                              ^
In file included from ../../nan/nan.h:54,
                 from ../src/bcrypt_node.cc:1:
../src/bcrypt_node.cc: At global scope:
/root/.cache/node-gyp/14.16.0/include/node/node.h:758:43: warning: cast between incompatible function types from 'void (*)(Nan::ADDON_REGISTER_FUNCTION_ARGS_TYPE)' {aka 'void (*)(v8::Local<v8::Object>)'} to 'node::addon_register_func' {aka 'void (*)(v8::Local<v8::Object>, v8::Local<v8::Value>, void*)'} [-Wcast-function-type]
  758 |       (node::addon_register_func) (regfunc),                          \
      |                                           ^
/root/.cache/node-gyp/14.16.0/include/node/node.h:792:3: note: in expansion of macro 'NODE_MODULE_X'
  792 |   NODE_MODULE_X(modname, regfunc, NULL, 0)  // NOLINT (readability/null_usage)
      |   ^~~~~~~~~~~~~
../src/bcrypt_node.cc:359:1: note: in expansion of macro 'NODE_MODULE'
  359 | NODE_MODULE(bcrypt_lib, init);
      | ^~~~~~~~~~~
  SOLINK_MODULE(target) Release/obj.target/bcrypt_lib.node
  COPY Release/bcrypt_lib.node
  COPY /app/node_modules/bcrypt/lib/binding/bcrypt_lib.node
  TOUCH Release/obj.target/action_after_build.stamp
make: Leaving directory '/app/node_modules/bcrypt/build'

> full-icu@1.3.1 postinstall /app/node_modules/full-icu
> node postinstall.js

Note: Your node was not compiled with the ‘small-icu’ case, so the ICU data is not customizable through this script. Exitting.

> nodemon@1.19.4 postinstall /app/node_modules/nodemon
> node bin/postinstall || exit 0

Love nodemon? You can now support the project via the open collective:
 > https://opencollective.com/nodemon/donate

npm WARN pwndoc-backend@0.0.1 No repository field.
npm WARN pwndoc-backend@0.0.1 No license field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 1632 packages from 1256 contributors and audited 1636 packages in 24.131s

19 packages are looking for funding
  run `npm fund` for details

found 4715 vulnerabilities (4708 low, 2 moderate, 5 high)
  run `npm audit fix` to fix them, or `npm audit` for details
Removing intermediate container ae57aa846aa3
 ---> d94afe8f6281
Step 7/11 : COPY . .
 ---> 9e6d18a333c1
Step 8/11 : EXPOSE 4242
 ---> Running in 5792a6f5a5ae
Removing intermediate container 5792a6f5a5ae
 ---> 6dfdacd30a32
Step 9/11 : ENV NODE_ENV prod
 ---> Running in 9aef85070f7a
Removing intermediate container 9aef85070f7a
 ---> b037a5d125bd
Step 10/11 : ENV NODE_ICU_DATA=node_modules/full-icu
 ---> Running in f8f750337ca7
Removing intermediate container f8f750337ca7
 ---> bc0aa982a4f4
Step 11/11 : ENTRYPOINT ["npm", "start"]
 ---> Running in d3efe1f1b9d9
Removing intermediate container d3efe1f1b9d9
 ---> d2e19d73ba52
Successfully built d2e19d73ba52
Successfully tagged yeln4ts/pwndoc:backend
Building pwndoc-frontend
Step 1/13 : FROM node:lts-alpine AS build
 ---> e2cefec76a61
Step 2/13 : WORKDIR /app
 ---> Running in a3a59d5c1a7d
Removing intermediate container a3a59d5c1a7d
 ---> 822b37a1c299
Step 3/13 : COPY package.json ./
 ---> bc6821c1a268
Step 4/13 : RUN npm install
 ---> Running in 814e183cac95
npm WARN deprecated axios@0.18.1: Critical security vulnerability fixed in v0.21.1. For more information, see https://github.com/axios/axios/pull/3410
npm WARN deprecated highlight.js@9.18.5: Support has ended for 9.x series. Upgrade to @latest
npm WARN deprecated fsevents@2.1.3: "Please update to latest v2.3 or v2.2"
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated coffee-script@1.12.7: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated

> core-js@3.9.1 postinstall /app/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> https://opencollective.com/core-js 
> https://www.patreon.com/zloirock 

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)


> highlight.js@9.18.5 postinstall /app/node_modules/devtron/node_modules/highlight.js
> node deprecated.js

-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

  Verion 9 of Highlight.js has reached EOL.  It will no longer
  be supported or receive security updates in the future.
  Please upgrade to version 10 or encourage your indirect
  dependencies to do so.

  For more info:
  
  https://github.com/highlightjs/highlight.js/issues/2877
  https://github.com/highlightjs/highlight.js/blob/master/VERSION_10_UPGRADE.md
 
-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

> ejs@2.7.4 postinstall /app/node_modules/ejs
> node ./postinstall.js

Thank you for installing EJS: built with the Jake JavaScript build tool (https://jakejs.com/)


> electron@2.0.18 postinstall /app/node_modules/electron
> node install.js

Downloading SHASUMS256.txt
[============================================>] 100.0% of 5.39 kB (5.39 kB/s)
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.2.7 (node_modules/watchpack-chokidar2/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.2.7 (node_modules/webpack-dev-server/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 1766 packages from 1071 contributors and audited 1770 packages in 48.548s

105 packages are looking for funding
  run `npm fund` for details

found 2 vulnerabilities (1 moderate, 1 high)
  run `npm audit fix` to fix them, or `npm audit` for details
Removing intermediate container 814e183cac95
 ---> 37c2e0039bea
Step 5/13 : COPY . .
 ---> 7bae14289399
Step 6/13 : RUN npm run build
 ---> Running in 493ed0cf443e

> pwndoc-frontend@1.0.0 build /app
> quasar build


 Build mode........ spa
 Pkg quasar........ v1.15.4
 Pkg @quasar/app... v2.1.15
 Debugging......... no
 Publishing........ no

 Configured browser support (at least 91.04% of global marketshare):
 · Chrome for Android >= 88
 · Firefox for Android >= 85
 · QQ for Android >= 10.4
 · UC for Android >= 12.12
 · Android >= 81
 · Baidu >= 7.12
 · Bb >= 7
 · Chrome >= 87
 · Edge >= 87
 · Firefox >= 84
 · IE >= 11
 · IE Mobile >= 10
 · iOS >= 13.4-13.7
 · Kaios >= 2.5
 · Opera Mini >= all
 · Opera Mobile >= 12.1
 · Opera >= 72
 · Safari >= 13.1
 · Samsung >= 12.0

 App · Reading quasar.conf.js
 App · Transpiling JS (Babel active) - includes IE11 support
 App · Extending SPA Webpack config
 App · Cleaned build artifact: "/app/dist/spa"
 App · Generating Webpack entry point
 App · Compiling with Webpack...
 App · Compiling SPA...
 App · Compiled SPA done in 26323 ms
 App · Added build artifact "/app/dist/spa"

 Build summary for css/js/json (the rest are omitted):
 ╔═══════════════════════════╤════════════╤═══════════╗
 ║                     Asset │       Size │   Gzipped ║
 ╟───────────────────────────┼────────────┼───────────╢
 ║            3.e07438fc.css │    0.06 KB │   0.07 KB ║
 ║            4.757e457b.css │    0.14 KB │   0.11 KB ║
 ║            5.752ddb70.css │    0.50 KB │   0.32 KB ║
 ║            6.6fd6a702.css │    0.04 KB │   0.06 KB ║
 ║            7.e22f5d2f.css │    0.31 KB │   0.19 KB ║
 ║            8.61228a19.css │    0.04 KB │   0.06 KB ║
 ║          app.b83631b8.css │    1.19 KB │   0.47 KB ║
 ║ chunk-common.29d186ab.css │    3.80 KB │   1.20 KB ║
 ║       vendor.4ce50180.css │  469.48 KB │  83.09 KB ║
 ╟───────────────────────────┼────────────┼───────────╢
 ║            10.c01e4382.js │    0.85 KB │   0.50 KB ║
 ║            11.0b44b022.js │    9.12 KB │   2.75 KB ║
 ║            12.4c2ddf6f.js │    8.84 KB │   2.42 KB ║
 ║            13.b458d06d.js │    8.86 KB │   3.04 KB ║
 ║            14.43decf18.js │    4.00 KB │   1.41 KB ║
 ║            15.c84461e8.js │    0.34 KB │   0.26 KB ║
 ║            16.d90dd7df.js │   13.75 KB │   3.96 KB ║
 ║            17.36e31fba.js │   12.96 KB │   2.50 KB ║
 ║            18.c7f77157.js │   12.39 KB │   2.48 KB ║
 ║            19.8114cd3a.js │    7.93 KB │   2.23 KB ║
 ║            20.6b804b8a.js │    7.19 KB │   2.28 KB ║
 ║            21.4b176b38.js │    1.86 KB │   0.69 KB ║
 ║            22.bf9fc108.js │    9.46 KB │   2.51 KB ║
 ║            23.b432912c.js │    4.20 KB │   1.29 KB ║
 ║             3.9998821a.js │    2.46 KB │   0.99 KB ║
 ║             4.21c03de3.js │   12.62 KB │   3.32 KB ║
 ║             5.4cfe4147.js │    9.19 KB │   3.03 KB ║
 ║             6.66aed60d.js │   38.21 KB │   5.65 KB ║
 ║             7.6f9a2376.js │    5.81 KB │   1.64 KB ║
 ║             8.77ce5bdf.js │   42.67 KB │   6.67 KB ║
 ║             9.61db5f81.js │    0.85 KB │   0.50 KB ║
 ║           app.a05c288b.js │   12.82 KB │   4.78 KB ║
 ║  chunk-common.1f0ae542.js │   41.94 KB │  11.05 KB ║
 ║             cvsscalc30.js │   11.49 KB │   3.22 KB ║
 ║        vendor.f2fef5fa.js │ 1360.10 KB │ 416.77 KB ║
 ╚═══════════════════════════╧════════════╧═══════════╝
 

 App · ⚠️  1 warning encountered:

chunk chunk-common [mini-css-extract-plugin]
Conflicting order. Following module has been added:
 * css ./node_modules/css-loader/dist/cjs.js??ref--7-oneOf-2-1!./node_modules/vue-loader/lib/loaders/stylePostLoader.js!./node_modules/postcss-loader/src??ref--7-oneOf-2-2!./node_modules/stylus-loader??ref--7-oneOf-2-3!./node_modules/@quasar/app/lib/webpack/loader.quasar-stylus-variables.js!./node_modules/@quasar/app/lib/webpack/loader.auto-import-client.js?kebab!./node_modules/vue-loader/lib??vue-loader-options!./src/components/breadcrumb.vue?vue&type=style&index=0&id=43e72eac&lang=stylus&scoped=true&
despite it was not able to fulfill desired ordering with these modules:
 * css ./node_modules/css-loader/dist/cjs.js??ref--8-oneOf-2-1!./node_modules/vue-loader/lib/loaders/stylePostLoader.js!./node_modules/postcss-loader/src??ref--8-oneOf-2-2!./node_modules/sass-loader/dist/cjs.js??ref--8-oneOf-2-3!./node_modules/@quasar/app/lib/webpack/loader.quasar-scss-variables.js!./node_modules/@quasar/app/lib/webpack/loader.auto-import-client.js?kebab!./node_modules/vue-loader/lib??vue-loader-options!./src/components/editor.vue?vue&type=style&index=0&lang=scss&
   - couldn't fulfill desired order of chunk group(s) , , 


 App · ⚠️  Build succeeded, but with 1 warning. Check log above.

 Build succeeded

 Build mode........ spa
 Pkg quasar........ v1.15.4
 Pkg @quasar/app... v2.1.15
 Debugging......... no
 Publishing........ no
 Transpiled JS..... yes (Babel) - includes IE11 support
 ==================
 Output folder..... /app/dist/spa

 Tip: Built files are meant to be served over an HTTP server
      Opening index.html over file:// won't work

 Tip: You can use "$ quasar serve" command to create a web server,
      both for testing or production. Type "$ quasar serve -h" for
      parameters. Also, an npm script (usually named "start") can
      be added for deployment environments.
      If you're using Vue Router "history" mode, don't forget to
      specify the "--history" parameter: "$ quasar serve --history"

Removing intermediate container 493ed0cf443e
 ---> ce0c87e27e15
Step 7/13 : FROM nginx:stable-alpine
stable-alpine: Pulling from library/nginx
e95f33c60a64: Already exists
6b9066ff94f0: Pull complete
d00048cae6c8: Pull complete
673a80f76512: Pull complete
5265c6a8bcaa: Pull complete
Digest: sha256:34039e81cf9de5f7f92f6280701e92cd51b85fb6b7170c41f6bf8920fcc79f8e
Status: Downloaded newer image for nginx:stable-alpine
 ---> b51b391571c1
Step 8/13 : COPY .docker/nginx.conf /etc/nginx/conf.d/default.conf
 ---> 83b2cef0a212
Step 9/13 : RUN mkdir -p /etc/nginx/ssl
 ---> Running in 35c23c6b293a
Removing intermediate container 35c23c6b293a
 ---> 6b8868731c4e
Step 10/13 : COPY ssl/server* /etc/nginx/ssl/
 ---> cf38fc9909bb
Step 11/13 : COPY --from=build /app/dist/spa /usr/share/nginx/html
 ---> 03a6f79380ac
Step 12/13 : EXPOSE 80
 ---> Running in 2e8d542303ec
Removing intermediate container 2e8d542303ec
 ---> e463bc46b32e
Step 13/13 : CMD ["nginx", "-g", "daemon off;"]
 ---> Running in 078fc96ac896
Removing intermediate container 078fc96ac896
 ---> ce54cef0e296
Successfully built ce54cef0e296
Successfully tagged yeln4ts/pwndoc:frontend
Creating mongo-pwndoc    ... done
Creating pwndoc-frontend ... done
Creating pwndoc-backend  ... done

```
